sandbox/settings.py

```diff
@@ -594,6 +612,11 @@ def post_setup(cls):
             with sentry_sdk.configure_scope() as scope:
                 scope.set_extra("application", "backend")
 
+        # If a Joanie Backend has been configured, we add it into LMS_BACKENDS dict
+        if cls.JOANIE_BACKEND.get('BASE_URL') is not None:
+            cls.RICHIE_LMS_BACKENDS.append({ 'NAME': 'joanie', **cls.JOANIE_BACKEND })
```

Comment:
in order to avoid adding this "joanie" name, you could have a property "is_joanie = True" on the JoanieBackend so when you import the path of the backend given in JOANIE_BACKEND["BACKEND"], you will know it is one?

Reply:
Yep indeed. But in this way, I have to import and instantiate each LMS backends configured to check if this is Joanie one.

Reply:
it can be a class attribute so you only need to import but not instantiate it.